WcaOnRails/app/views/competitions/show_all_results.html.erb

```diff
@@ -22,6 +23,15 @@
         <% end %>
       <% end %>
     <% end %>
-
+    <div class="event-selector-bottom text-center">
+      <% @competition.events_with_round_types_with_results.each do |event, rounds_with_results| %>
```

Comment:
Instead of repeating the code, I think it would be better to extract this into a partial.